token/program-2022/src/extension/token_metadata/processor.rs

```diff
@@ -142,9 +142,26 @@ pub fn process_remove_key(
 /// Processes a [UpdateAuthority](enum.TokenMetadataInstruction.html) instruction.
 pub fn process_update_authority(
     _program_id: &Pubkey,
-    _accounts: &[AccountInfo],
-    _data: UpdateAuthority,
+    accounts: &[AccountInfo],
+    data: UpdateAuthority,
 ) -> ProgramResult {
+    let account_info_iter = &mut accounts.iter();
+    let metadata_info = next_account_info(account_info_iter)?;
+    let update_authority_info = next_account_info(account_info_iter)?;
+
+    // deserialize the metadata, but scope the data borrow since we'll probably
+    // realloc the account
```

Comment:
nit: This comment doesn't make sense if we never need to realloc

Reply:
Ah nice call, thanks!